internal/mode/static/nginx/conf/nginx.conf

```diff
@@ -1,4 +1,5 @@
 load_module /usr/lib/nginx/modules/ngx_http_js_module.so;
+include /etc/nginx/includes/*.conf;
```

Comment:
Tried to include the specific file, but it must exist beforehand in order to nginx to start. Can't create it in the Dockerfile because the emptyDir volume mount overwrites it.